lib/queue.js

```diff
@@ -643,7 +643,7 @@ class Queue extends Emitter {
 
           /* istanbul ignore else */
           if (results) {
-            const status = results[0], result = results[1];
+            const [status, result] = results;
```

Comment:
I don't remember exactly, but I think `bee-queue` might still purport to support Node 4.0.0 - this may be a breaking change as the destructuring assignment isn't supported until 6.0.0.